server.go

```diff
@@ -75,9 +83,39 @@ func NewServer(o Options) (*Server, error) {
 			return
 		}
 
+		q := r.Question[0]
+
 		// Filter Questions:
 		if r.Question = s.filter(r.Question); len(r.Question) == 0 {
-			w.WriteMsg(r)
+			IPQuery := s.isIPQuery(q)
+			m := new(dns.Msg)
+			m.SetReply(r)
+
+			switch IPQuery {
+			case _IP4Query:
+				rr_header := dns.RR_Header{
+					Name:   q.Name,
+					Rrtype: dns.TypeA,
+					Class:  dns.ClassINET,
+					Ttl:    600,
+				}
+
+				a := &dns.A{rr_header, net.ParseIP("192.168.1.117").To4()}
```

Comment:
You can't just use this random IP because it may be an actual IP to some user later.  In an ideal world I would think we would just return an `NXDOMAIN` result.